File: app/src/net/reichholf/dreamdroid/parsers/enigma2/saxhandler/E2TagHandler.java
/* © 2010 Stephan Reichholf <stephan at reichholf dot net>
 * 
 * Licensed under the Create-Commons Attribution-Noncommercial-Share Alike 3.0 Unported
 * http://creativecommons.org/licenses/by-nc-sa/3.0/
 */

package net.reichholf.dreamdroid.parsers.enigma2.saxhandler;

public class E2TagHandler extends E2SimpleListHandler {
	protected static final String TAG_E2TAG = "e2tag";

	public E2TagHandler(){
		super(TAG_E2TAG);
	}
}

File: app/src/net/reichholf/dreamdroid/parsers/enigma2/saxhandler/E2TagHandler.kt
/* © 2010 Stephan Reichholf <stephan at reichholf dot net>
 *
 * Licensed under the Create-Commons Attribution-Noncommercial-Share Alike 3.0 Unported
 * http://creativecommons.org/licenses/by-nc-sa/3.0/
 */

package net.reichholf.dreamdroid.parsers.enigma2.saxhandler

class E2TagHandler : E2SimpleListHandler("e2tag") {
    companion object {
        @JvmField
        protected val TAG_E2TAG: String = "e2tag"
    }
}
